app/shared/controllers/RequestHandler.scala

```diff
@@ -70,6 +70,9 @@ object RequestHandler {
     def withAuditing(auditHandler: AuditHandler): RequestHandlerBuilder[Input, Output] =
       copy(auditHandler = Some(auditHandler))
 
+    def withResponseModifier(modelHandler: Output => Output): RequestHandlerBuilder[Input, Output] =
```

Comment:
could "modelHandler" be called "responseModifier" or sm?  ("modelHandler" doesn't really make sense)